any_member.hpp

```diff
@@ -596,25 +595,20 @@ any_member<ClassType>& MemberSymbolTable<ClassType>::operator[]
     (std::string const& s
     )
 {
-    auto i = map_.find(s);
-    if(map_.end() == i)
+    auto i = std::lower_bound(member_names_.begin(), member_names_.end(), s);
```

Comment:
Not sure why did I use `lower_bound()` here instead of `find()`... If you don't see any reason to do it neither, please replace it with the simpler `find()`.